rt-fs/main.go

```diff
@@ -21,5 +21,7 @@ func getApp() components.App {
 
```

Comment:
Let's change the version from v1.0.0 to v1.1.0
Let's also update the release notes.